2_observations.yml

```diff
@@ -0,0 +1,76 @@
+target_default: 2_observations
+
+packages:
+  - sf
+  - tidyverse
+  - lwgeom
+  - smoothr
+  - igraph
+  - purrr
+
+sources:
+  - 2_observations/src/crosswalk_functions.R
+  - 2_observations/src/data_munge_functions.R
+
+targets:
+  2_observations:
+    depends:
+     - 2_observations/out/basin_temp_data.rds.ind
+     
+  # temperature sites
+  2_observations/in/all_sites.rds:
+    command: gd_get('2_observations/in/all_sites.rds.ind')
+    
+  # munged temperature data
+  2_observations/in/daily_temperatures.rds:
+    command: gd_get('2_observations/in/daily_temperatures.rds.ind')
+     
+  2_observations/out/crosswalk_site_reach.rds.ind:
+    command: crosswalk_sites_to_reaches(
+      out_ind = target_name,
+      sites_ind = '2_observations/in/all_sites.rds.ind',
+      boundary_ind = '1_network/out/boundary.rds.ind',
+      network_ind = '1_network/out/network.rds.ind')
+    depends: 
+      - 2_observations/src/subset_closest.R
```

Comment:
Why did you go with a `depends` instead of declaring the R script in `sources`?

Reply:
The function contained in `subset_closest.R` is called within `crosswalk_sites_to_reaches` - so I thought this was the way to capture that dependency?

Reply:
If I put both functions in the same script and listed it in `sources`, would that dependency be captured?

Reply:
Gotchya. It is how I was doing it without scipiper, but I think we can go with just adding it to `sources` here instead - remake seems to be really clever about navigating down into nested functions.

Reply:
Yes, I think `sources` would capture the dependency. But now you've made me nervous, too =). You could test it here if this is a cheap function to run - after moving from `depends` to `sources`, just build this target, try modifying `subset_closest.R` by adding a print() or something, and see if the target becomes dirty. (If it does, that's good.)

Reply:
Will investigate!

Reply:
ok - glad I was wrong! listing in sources does capture the dependencies. Will modify this part and push.